tools/esdlcmd/esdl-publish.cpp

```diff
@@ -160,11 +160,11 @@ class EsdlPublishCmd : public EsdlPublishCmdCommon
     void usage()
     {
         printf("\nUsage:\n\n"
-                "esdl publish <servicename> <filename.ecm> [command options]\n\n"
-                "   <servicename>        The ESDL defined ESP service to publish\n"
-                "   <filename.ecm>       The ESDL file containing service definition\n"
+                "esdl publish <servicename> <filename.(ecm|esdl)> [command options]\n\n"
```

Comment:
It's fine to document this way, but as far as the user specified file I would think that any extension would be supported?  I.e. ".xxyyzz" should work?  We just don't have to advertise it.